codex-rs/core/src/codex.rs

```diff
@@ -1182,7 +1203,7 @@ async fn try_run_turn(
             }
         }
     }
-    Ok(output)
+    // unreachable: loop only exits via return statements above
```

Comment:
can remove this comment; the compiler would flag any added code as unreachable, right?